plugins/polio/export_utils.py

```diff
@@ -0,0 +1,114 @@
+from ctypes import alignment
+from curses import start_color
+from turtle import color
+from openpyxl import Workbook
+from openpyxl.writer.excel import save_virtual_workbook
+from openpyxl.utils import get_column_letter
+from openpyxl.styles import Alignment, Font, PatternFill
+from openpyxl.styles.borders import Border, Side
+import datetime as dt
+import calendar
+
+# from datetime import datetime
+
+
+def generate_xlsx(filename, datas):
```

Comment:
Iaso generate multiple xlsx for different things, so I think it would be great to use a more verbose name (generate_xlsx_campaigns_calendar for example)